### PR TITLE
Add onboarding wizard for Remix SDK

### PR DESCRIPTION
**Pull request by AbhiPrasad**
ref: https://github.com/getsentry/sentry-javascript/issues/4894

### CODE REVIEW

**Review by Neo-Zhixing — APPROVED** [review]
Looks good on the growth side but I'll let others comment on the content

File: src/wizard/javascript/remix.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+```bash
+# Using yarn
+yarn add @sentry/remix
+
+# Using npm
+npm install --save @sentry/remix
+```
+
+Next, import and initialize initialize Sentry in your Remix entry points for both the client and server.
```
Comment (imatwawana) [inline src/wizard/javascript/remix.md:18]:
```suggestion
Next, import and initialize initialize Sentry in your Remix entry points for both the client and server:
```

File: src/wizard/javascript/remix.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        useEffect,
+        useLocation,
+        useMatches,
+      ),
+    }),
+  ],
+});
+```
+
+Initialize Sentry in your entry point for the server to capture exceptions and get performance metrics for your [`action`](https://remix.run/docs/en/v1/api/conventions#action) and [`loader`](https://remix.run/docs/en/v1/api/conventions#loader) functions. You can also initialize Sentry's database integrations, such as Prisma, to get spans for your database calls.
```
Comment (imatwawana) [inline src/wizard/javascript/remix.md:40]:
```suggestion
Initialize Sentry in your entry point for the server to capture exceptions and get performance metrics for your [`action`](https://remix.run/docs/en/v1/api/conventions#action) and [`loader`](https://remix.run/docs/en/v1/api/conventions#loader) functions. You can also initialize Sentry's database integrations, such as Prisma, to get spans for your database calls:
```

File: src/wizard/javascript/remix.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+import * as Sentry from "@sentry/remix";
+
+Sentry.init({
+  dsn: "__DSN__",
+  tracesSampleRate: 1,
+  integrations: [new Sentry.Integrations.Prisma({ client: prisma })],
+});
+```
+
+Finally, wrap your Remix root with `withSentry` to catch React component errors and to get parameterized router transactions.
```
Comment (imatwawana) [inline src/wizard/javascript/remix.md:54]:
```suggestion
Lastly, wrap your Remix root with `withSentry` to catch React component errors and to get parameterized router transactions:
```

File: src/wizard/javascript/remix.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+        <LiveReload />
+      </body>
+    </html>
+  );
+}
+
+export default withSentry(App);
+```
+
+Once you've verified the library is initialized properly and sent a test event, consider visiting our [complete Remix docs](https://docs.sentry.io/platforms/javascript/guides/remix/). There you'll find additional instructions for configuring the Remix SDK.
```
Comment (imatwawana) [inline src/wizard/javascript/remix.md:88]:
```suggestion
Once you've verified the library is initialized properly and sent a test event, consider visiting our [complete Remix docs](https://docs.sentry.io/platforms/javascript/guides/remix/). There, you'll find additional instructions for configuring the Remix SDK.
```

**Review by imatwawana — APPROVED** [review]

**Pull request merged**